src/h160_hexadecimal.rs

```diff
@@ -0,0 +1,70 @@
+use primitive_types::H160;
```

Comment:
This shouldn't be needed if the [`serde` feature is enabled](https://github.com/paritytech/parity-common/blob/c031b0971c3683ac9f2ab6b365064979afa24588/primitive-types/Cargo.toml#L24) in the `primitive-types` crate.